ext/tokenizer/tokenizer_data_arginfo.h

```diff
@@ -25,7 +25,7 @@ static void register_tokenizer_data_symbols(int module_number)
 	REGISTER_LONG_CONSTANT("T_LOGICAL_AND", T_LOGICAL_AND, CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("T_PRINT", T_PRINT, CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("T_YIELD", T_YIELD, CONST_PERSISTENT);
-	REGISTER_LONG_CONSTANT("T_YIELD_FROM", T_YIELD_FROM, CONST_PERSISTENT);
+	REGISTER_LONG_CONSTANT("T_FROM", T_FROM, CONST_PERSISTENT);
```

Comment:
Should `T_YIELD_FROM` be deprecated first (as unused as of this PR) and removed in 9.0 ?

Reply:
That doesn't really apply for tokenizer constants. They always match what exists for the specific PHP version.
(It's also a generated file.)

Reply:
Fair enough, just makes the BC break larger again.